shopify-cli.gemspec

```diff
@@ -0,0 +1,33 @@
+require_relative 'lib/shopify-cli/version'
+
+Gem::Specification.new do |spec|
+  spec.name = "shopify-cli"
+  spec.version = ShopifyCli::VERSION
+  spec.authors = ["Shopify"]
+  spec.email = ["developers@jadedpixel.com"]
+  spec.license = 'Nonstandard'
+
+  spec.summary = "Shopify App CLI helps you build Shopify apps faster."
```

Comment:
Just call it Shopify CLI for now, I am hoping to change this over because we make more than just apps.